packages/webapp/src/routes/__tests__/land/check-and-submit.spec.js

```diff
@@ -48,7 +48,7 @@ describe(url, () => {
           expect(viewArgs[1].application.applicant.emailAddress).toEqual('john.smith@test.com')
           expect(viewArgs[1].application.applicant.contactId).toEqual('1234567890')
           // expect(viewArgs[1].application.applicant.role).toEqual('Other: test role')
-          expect(viewArgs[1].application.legalAgreementParties[1]).toEqual({ name: 'org2', role: 'Landowner' })
+          // expect(viewArgs[1].application.legalAgreementParties[1]).toEqual({ name: 'org2', role: 'Landowner' })
```

Comment:
check and submit changes not done. we have removed legalAgreementParties from legal agreement journey. commented to avoid unit tests failure. this test file will be updated as part of 3552